src/modules/auth/repository/authRepository.ts

```diff
@@ -32,4 +33,28 @@ const destroySessionByAttribute = async ({ primaryKey, primaryValue, secondaryKe
     await models.sessions.destroy({where: { [primaryKey]: primaryValue, [secondaryKey]: secondaryValue }});
 };
 
-export default { createUser, findUserByAttributes, updateUserByAttributes, createSession, findSessionByAttributes, findSessionByTripleAttributes, destroySessionByAttribute };
\ No newline at end of file
+const findAuthPlatformByAttributes = async ({ primaryKey, primaryValue, secondaryKey, secondaryValue }: FindByAttributesInterface): Promise<AuthManagementsInterface | null> => {
+    return await models.authManagements.findOne({ where: { [primaryKey]: primaryValue, [secondaryKey]: secondaryValue } });
```

Comment:
the props passed to the function should be the same as the ones used on the where clause [key] : value